docs/typographical-conventions.rst

```diff
@@ -98,10 +98,11 @@ ini:
 
 .. code-block:: ini
 
-    [nosetests]
-    match=^test
-    where=pyramid
-    nocapture=1
+    [tool:pytest]
+    python_files = test_*.py
+    testpaths =
+        tests
+    addopts = -W always --cov --cov-report=term-missing
```

Comment:
drop --cov-report arg

Reply:
This is in the docs/typographical conventions. I'll drop it, but it's just an example.